tests/e2e/gc_block_volume_expansion.go

```diff
@@ -1628,6 +1639,291 @@ var _ = ginkgo.Describe("[csi-guest] Volume Expansion Test", func() {
 
 	})
 
+	/*
+	   This test verifies the static provisioning workflow in guest cluster when svcPVC=gcPVC.
+
+	   Test Steps:
+	   1. Create FCD with valid storage policy on gc-svc.
+	   2. Create Resource quota
+	   3. Create CNS register volume with above created FCD on SVC
+	   4. verify svc-PV, svc-PVC got created , check the bidirectional reference on svc.
+	   5. On GC create a gc-PV by pointing volume handle got created by static provisioning on svc-PVC (in step 4)
+	   6. On GC create a gc-PVC pointing to above created PV (step 5)
+	   7. Wait for gc-PV , gc-PVC to get bound
+	   8. Create POD, verify the status
+	   9. Trigger online volume expansion on gc-pvc and make sure volume expansion is successful.
+	   9. Delete all the above created PV, PVC and resource quota
+	*/
+	ginkgo.It("Online volume resize on statically created PVC on guest cluster svcPVC=gcPVC", func() {
+		var err error
+		ctx, cancel := context.WithCancel(context.Background())
+		defer cancel()
+
+		curtime := time.Now().Unix()
+		randomValue := rand.Int()
+		val := strconv.FormatInt(int64(randomValue), 10)
+		val = string(val[1:3])
+		curtimestring := strconv.FormatInt(curtime, 10)
+		svpvcName := "cns-pvc-" + curtimestring + val
+		framework.Logf("pvc name :%s", svpvcName)
+		namespace = getNamespaceToRunTests(f)
+
+		_, storageclass, profileID := staticProvisioningPreSetUpUtil(ctx, f, client, storagePolicyName)
+
+		// get supvervisor cluster client
+		svcClient, svNamespace := getSvcClientAndNamespace()
+
+		// Get restConfig
+		var restConfig *restclient.Config
+		if k8senv := GetAndExpectStringEnvVar("SUPERVISOR_CLUSTER_KUBE_CONFIG"); k8senv != "" {
+			restConfig, err = clientcmd.BuildConfigFromFlags("", k8senv)
+			gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		}
```

Comment:
Put this in a util method and initialize once. Refer `getSvcClientAndNamespace()`.

Reply:
Added getRestConfigClient()  method in util and initialising it in Before each